### DISCUSSION TITLE
版本文件丢失？

### DISCUSSION THREAD

**Discussion by DevilPPPP** [Meloong-Git/PCL#3256] [提问]
电脑未关机，重新登录PCL2启动器后发现所有之前的版本文件全部丢失，当我重新下载版本之后，反而出现了之前的旧存档，mod已找到，主要是存档，如何找到最新存档？

**Comment by allMagicNB** [comment]
你确定你用 Everything 搜索了？

**Reply by DevilPPPP** [reply]
搜索了，仍然找不到，即便是用其他软件找了删除了的文件也没找到

**Reply by DevilPPPP** [reply]
目前已经用了Everthing和数据修复软件均未找到

**Reply by allMagicNB** [reply]
> 目前已经用了Everthing和数据修复软件均未找到

日志传下？

**Reply by DevilPPPP** [reply]
错误日志还是启动器的，错误日志他没有弹出

**Reply by allMagicNB** [reply]
> 错误日志还是启动器的，错误日志他没有弹出

就是启动器日志

**Reply by DevilPPPP** [reply]
只有版本文件被删除后6小时左右的日志，以前的找不到在哪里

**Reply by DevilPPPP** [reply]
只有游戏日志，并且发现旧存档（所需要找的存档旧版，相当于回档了）里面的文件丢失了很多
[2024-02-03-1.log.gz](https://github.com/Hex-Dragon/PCL2/files/14155860/2024-02-03-1.log.gz)
[2024-02-03-2.log.gz](https://github.com/Hex-Dragon/PCL2/files/14155861/2024-02-03-2.log.gz)

**Reply by allMagicNB** [reply]
> 只有游戏日志，并且发现旧存档（所需要找的存档旧版，相当于回档了）里面的文件丢失了很多
> [2024-02-03-1.log.gz](https://github.com/Hex-Dragon/PCL2/files/14155860/2024-02-03-1.log.gz)
> [2024-02-03-2.log.gz](https://github.com/Hex-Dragon/PCL2/files/14155861/2024-02-03-2.log.gz)
> 

PCL2日志呢

**Reply by DevilPPPP** [reply]
只有已经被删后的日志，以前的日志找不到
[Log1.txt](https://github.com/Hex-Dragon/PCL2/files/14156064/Log1.txt)
[Log2.txt](https://github.com/Hex-Dragon/PCL2/files/14156065/Log2.txt)
[Log3.txt](https://github.com/Hex-Dragon/PCL2/files/14156066/Log3.txt)
[Log4.txt](https://github.com/Hex-Dragon/PCL2/files/14156067/Log4.txt)
[Log5.txt](https://github.com/Hex-Dragon/PCL2/files/14156068/Log5.txt)

**Comment by DevilPPPP** [comment]
搜索后未找到解决方案，麻烦您告诉我如何解决

**Reply by allMagicNB** [reply]
> 搜索后未找到解决方案，麻烦您告诉我如何解决

我说，使用Everything工具搜索
[这里](https://www.voidtools.com/zh-cn/)是它的官网

**Reply by DevilPPPP** [reply]
找不到存档

**Comment by ghost** [comment]
> 电脑未关机，重新登录PCL2启动器后发现所有之前的版本文件全部丢失，当我重新下载版本之后，反而出现了之前的旧存档，mod已找到，主要是存档，如何找到最新存档？

用数据恢复软件扫一下

**Reply by DevilPPPP** [reply]
扫了，没找到

**Comment by ghost** [comment]
> 扫了，没找到

应该是FTL表没了

**Reply by ghost** [reply]
FTL是啥

**Reply by allMagicNB** [reply]
> FTL是啥
> 

闪存转换层？

**Reply by DevilPPPP** [reply]
应该不是吧，其他文件都好好地

**Reply by ghost** [reply]
> 应该不是吧，其他文件都好好地

因为有旧的FTL表，他在装 Mod 前没存档，装好后有了旧存档，说明SSD的主控芯片通过旧的FTL表修复了SSD上的数据

**Reply by DevilPPPP** [reply]
那版本文件在重新打开都没了是一样的道理？那咋办，只是想要回存档

**Reply by DevilPPPP** [reply]
所以怎么办

**Reply by ghost** [reply]
废了，就算扫到了也是.chk（恢复的文件碎片），没有metadata根本还原不了
